TradingBot/Models/TradingAction.cs

```diff
@@ -129,7 +129,7 @@ public static TradingAction MarketSell(TradingSymbol symbol, decimal quantity, D
             Id = Guid.NewGuid(),
             CreatedAt = createdAt,
             Quantity = quantity,
-            Price = price,
+            Price = Math.Round(price, 2),
```

Comment:
I think we should round up for buy actions and down for sell actions. Some strategies try to use all available money to buy/sell something, and rounding the other way may mean that there is not enough money to place the order.